source/multigrid/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 ## ---------------------------------------------------------------------
 ##
-## Copyright (C) 2012 - 2013 by the deal.II authors
+## Copyright (C) 2012 - 2013, 2015 by the deal.II authors
```

Comment:
Would you allow me to do this as a follow-up? In fact, I think we should always tag every file's copyright notice for every release.